### PR TITLE
man: fix description for rootfs

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao mashimiao.fnst@cn.fujitsu.com

### CODE REVIEW

File: man/ocitools-generate.1.md [hunk 1]
```diff
@@ -168,7 +168,9 @@ read the configuration from `config.json`.
   Values are "shared, rshared, private, rprivate, slave, rslave"
 
 **--rootfs**=ROOTFSPATH
-  Path to the rootfs
+  Path to the rootfs, which is relative to bundle path.
+  e.g the absolute path of rootfs is /to/bundle/rootfs, bundle path is /to/bundle,
+  then the value set as ROOTFSPATH should be `rootfs`.
```
Comment (wking) [inline man/ocitools-generate.1.md]: This is useful information, but I think we want to land it in runtime-spec first.  Currently [runtime-spec is fairly weak on how relative paths should be handled][1], although relative path semantics for `cgroupsPath` have [become more clear recently][2].

[1]: https://github.com/opencontainers/runtime-spec/issues/509#issuecomment-228414738
[2]: https://github.com/opencontainers/runtime-spec/pull/493/files#diff-428eec4013a655816cdefafd5d3505f1R164

**Comment by Mashimiao** [comment]
PR rebased.
Ping @mrunalp  @liangchenye

**Comment by liangchenye** [comment]
@Mashimiao 
You may need to change this according to new explaination:
https://github.com/opencontainers/runtime-spec/blob/master/config.md#root-configuration

**Comment by liangchenye** [comment]
Relevant PR 
https://github.com/opencontainers/runtime-tools/pull/233

**Comment by Mashimiao** [comment]
If need to change as spec, we have to wait #233  to be merged. It matches the current implementation.

**Comment by Mashimiao** [comment]
@liangchenye  @mrunalp   fixed and rebased.

**Comment by wking** [comment]
aa87e86e0 looks good to me.  It skirts the current bundle requirement
that I'm trying to lift with the in-flight
opencontainers/runtime-spec#469, but we can tighten down on that in a
subsequent PR.

**Comment by Mashimiao** [comment]
rebased. @opencontainers/runtime-tools-maintainers  PTAL

**Comment by hqhq** [comment]
LGTM

**Comment by liangchenye** [comment]
LGTM

**Pull request merged**